presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -222,6 +222,18 @@ the Java `SimpleDateFormat`_ pattern format.
 
     Parses ``string`` into a timestamp with time zone using ``format``.
 
+.. function:: unix_timestamp() -> bigint
+
+    Returns a unix timestamp - the number of seconds since Unix Epoch (1970.01.01 00:00:00)
+
+.. function:: unix_timestamp(string) -> bigint
```

Comment:
Agreed, I put it in this section because I treat `unix_timestamp` as a single function, with 3 overloads. Shall I put the two overloads not using date formats in 'Date and Time Functions' section?